utils/test_harness/sysman/include/test_harness_sysman_power.hpp

```diff
@@ -17,28 +17,28 @@ std::vector<zes_pwr_handle_t> get_power_handles(zes_device_handle_t device,
                                                 uint32_t &count);
 zes_pwr_handle_t get_card_power_handle(zes_device_handle_t device);
 zes_power_properties_t get_power_properties(zes_pwr_handle_t pPowerhandle);
-void get_power_limits(zes_pwr_handle_t pPowerHandle,
+ze_result_t get_power_limits(zes_pwr_handle_t pPowerHandle,
                       zes_power_sustained_limit_t *pSustained,
                       zes_power_burst_limit_t *pBurst,
                       zes_power_peak_limit_t *pPeak);
-void set_power_limits(zes_pwr_handle_t pPowerHandle,
+ze_result_t set_power_limits(zes_pwr_handle_t pPowerHandle,
                       zes_power_sustained_limit_t *pSustained,
                       zes_power_burst_limit_t *pBurst,
                       zes_power_peak_limit_t *pPeak);
 void get_power_energy_counter(zes_pwr_handle_t pPowerHandle,
                               zes_power_energy_counter_t *pEnergy);
-zes_energy_threshold_t
-get_power_energy_threshold(zes_pwr_handle_t pPowerHandle);
+ze_result_t get_power_energy_threshold(zes_pwr_handle_t pPowerHandle,
+                                       zes_energy_threshold_t *pThreshold);
 void set_power_energy_threshold(zes_pwr_handle_t pPowerHandle,
                                 double threshold);
-std::vector<zes_power_limit_ext_desc_t>
-get_power_limits_ext(zes_pwr_handle_t hPower, uint32_t *pCount);
-void set_power_limits_ext(zes_pwr_handle_t hPower, uint32_t *pCount,
+ze_result_t get_power_limits_ext(zes_pwr_handle_t hPower, uint32_t *pCount,
+    std::vector<zes_power_limit_ext_desc_t> &pPowerLimitsDescriptors);
+ze_result_t set_power_limits_ext(zes_pwr_handle_t hPower, uint32_t *pCount,
                           zes_power_limit_ext_desc_t *pSustained);
 void compare_power_descriptor_structures(
     zes_power_limit_ext_desc_t firstDescriptor,
     zes_power_limit_ext_desc_t secondDescriptor);
-uint32_t get_power_limit_count(zes_pwr_handle_t hPower);
+ze_result_t get_power_limit_count(zes_pwr_handle_t hPower, uint32_t *pCount);
```

Comment:
better to add a new overloaded function than modifying existing interface?, thoughts?

Reply:
only power module is calling these power specific harness methods. If we overload original methods will not be used anymore anywhere.